lib/cartopy/tests/mpl/test_quiver.py

```diff
@@ -0,0 +1,82 @@
+# (C) British Crown Copyright 2015, Met Office
+#
+# This file is part of cartopy.
+#
+# cartopy is free software: you can redistribute it and/or modify it under
+# the terms of the GNU Lesser General Public License as published by the
+# Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cartopy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with cartopy.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import (absolute_import, division, print_function)
+
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+
+from nose.tools import raises, assert_equal
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.testing.decorators import cleanup
+
+import cartopy.crs as ccrs
+
+
+# Note, other tests for quiver exist in test_mpl_integration.
+
+class TestQuiverShapes(unittest.TestCase):
+    @cleanup
```

Comment:
I don't think it makes sense to use the `cleanup` decorator here; it seems like it would clean up right after setup, but actually, it seems like it changes things sufficiently that `unittest` doesn't even call `setUp`.

I think maybe you should subclass from `CleanupTestCase`.